wolfcrypt/src/aes.c

```diff
@@ -7959,10 +7954,6 @@ int wc_AesXtsDecrypt(XtsAes* xaes, byte* out, const byte* in, word32 sz,
         return BAD_FUNC_ARG;
     }
 
-    if (in == NULL && sz > 0) {
```

Comment:
@JacobBarthelmeh : Did you intend for this function to allow a NULL input if the size is 0?

Reply:
In the case that sz is 0 it should error out farther down with a debug message.  I think this refactor is ok.